sat_cty/utils.py

```diff
@@ -6,6 +6,7 @@
 from os import makedirs
 import logging
 
+
```

Comment:
I should add https://pypi.org/project/black/ and https://pre-commit.com/ to this project to avoid ever having to manually format stuff.

Reply:
Woah, a conversation... How does this work?